commerce_coordinator/apps/commercetools/catalog_info/constants.py

```diff
@@ -72,6 +72,21 @@ class TwoUKeys:
     ORDER_NUMBER_CUSTOM_OBJECT_KEY = 'orderNumber'
 
 
+class CourseModes:
+    """Course Mode keys for use in Commercetools"""
+
+    # Course Modes
+    AUDIT = 'audit'
+    VERIFIED = 'verified'
+    PROFESSIONAL = 'professional'
+    CREDIT = 'credit'
+    HONOR = 'honor'
+    MICRO_CREDENTIAL = 'micro-credential'
+    MICROBACHELOR = 'microbachelor'
+    MICRO_MASTER = 'micro-master'
+    MICRO_CREDENTIAL_VERIFIED = 'micro-credential-verified'
```

Comment:
Should we only add the ones synced to commercetools? This give the impression that products exist with all types of mode. But let me know your opinion.

```suggestion
class CourseModes:
    """Course Mode keys for use in Commercetools"""

    VERIFIED = 'verified'
    PROFESSIONAL = 'professional'
    CREDIT = 'credit'
```

Reply:
i agree but we also want to stay decoupled edx system as much as we can. The Commercetools or CT aim to be the ecommerce solution for all types of products sources, not just discovery.